### PR TITLE
MTDSA-15634: Add STATEFUL to all endpoints - Individuals Charges API

### PR DESCRIPTION
**Pull request by beckfordp**

### CODE REVIEW

File: resources/public/api/conf/2.0/createAmendPensionCharges.yaml [hunk 1]
```diff
@@ -16,6 +16,12 @@ description: |
                 <td><p>Simulates a success response.</p></td>
            </tr> 
         </tbody>
+        <tbody>
+           <tr>
+                <td>STATEFUL</p></td>
+                <td><p>SSimulates a stateful success response.</p></td>
```
Comment (LouiseWong98) [inline resources/public/api/conf/2.0/createAmendPensionCharges.yaml:22]: typo in the word "Simulates"

File: resources/public/api/conf/2.0/deletePensionCharges.yaml [hunk 2]
```diff
@@ -20,6 +20,12 @@ description: |
                  <td><p>Simulates a scenario where the remote endpoint has indicated that no data can be found for the given period.</p></td>
             </tr>                     
         </tbody>
+        <tbody>
+            <tr>
+                  <td>STATEFUL</p></td>
+                  <td><p>SSimulates a stateful success response.</p></td>
```
Comment (LouiseWong98) [inline resources/public/api/conf/2.0/deletePensionCharges.yaml:26]: same as above

File: resources/public/api/conf/2.0/retrievePensionCharges.yaml [hunk 3]
```diff
@@ -20,6 +20,12 @@ description: |
               <td><p>Simulates a scenario where no data can be found.</p></td>
           </tr>
       </tbody>
+      <tbody>
+          <tr>
+              <td>STATEFUL</p></td>
+              <td><p>SSimulates a stateful success response.</p></td>
```
Comment (LouiseWong98) [inline resources/public/api/conf/2.0/retrievePensionCharges.yaml:26]: same as above

**Review by LouiseWong98 — DISMISSED** [review]

File: resources/public/api/conf/2.0/createAmendPensionCharges.yaml [hunk 4]
```diff
@@ -16,6 +16,12 @@ description: |
                 <td><p>Simulates a success response.</p></td>
            </tr> 
         </tbody>
+        <tbody>
```
Comment (rhysconnor) [inline resources/public/api/conf/2.0/createAmendPensionCharges.yaml:19]: I dont think we need the `tbody` tag here, we can just use the `table row` tag

**Review by mashukreza — APPROVED** [review]

**Review by mashukreza — APPROVED** [review]

**Pull request merged**